model-optimizer/automation/package_BOM.txt

```diff
@@ -534,7 +534,6 @@ extensions/middle/MulFakeQuantizeFuse.py
 extensions/middle/MXNetRNNSequenceNormalize.py
 extensions/middle/MXNetSplitMultiLayers.py
 extensions/middle/MXTileReplacer.py
-extensions/middle/NasNet.py
```

Comment:
The pattern in the NasNet transformation includes the StridedSlice operation. Will StridedSlice be generated with the MO after this change? Will MO insert additional transpose because of the StridedSlice? Can we run a benchmark to see the performance results after this change?

Reply:
In the NasNet we transformed StridedSlice to Crop, but then at the back phase we make it StridedSlice again. After removing NasNet transformation we avoid this useless conversion to Crop.

Reply:
Benchmark run for NasNet modes is in progress since yesterday. I'll update this PR when it finished.

Reply:
According to benchmark results NasNet models have the same load/read/execution time with this changes as on master branch.